News Theme Development.md

```diff
@@ -129,6 +129,36 @@ Common phrases needed across multiple blocks are declared in global-phrases-bloc
 
 ## Development Process
 
+### Release scripts (ran in github actions)
+
+#### `npm run release:canary` 
+
+- Only publish packages to canary tag
+- Publish packages based on only changes
+- Iterates the prerelease version using `--canary` arg that allows prerelease (not necessarily canary)
+
+
+#### `npm run release:beta` 
+
+- Only publish packages to beta
+- Publish packages based on only changes
+- Iterates the prerelease version using `--canary` arg that allows prerelease (not necessarily canary)
+
+
+#### `npm run release:rc` 
+
+- Only publish packages to rc
+- Publish packages based on only changes
+- Iterates the prerelease version using `--canary` arg that allows prerelease (not necessarily canary)
```

Comment:
Shouldn't this read `--rc`

Reply:
added explanation https://github.com/WPMedia/fusion-news-theme-blocks/pull/993/files#r674833117